src/core/qgshttptransaction.cpp

```diff
@@ -189,6 +197,9 @@ bool QgsHttpTransaction::getSynchronously( QByteArray &respondedContent, int red
   connect( http,   SIGNAL( stateChanged( int ) ),
            this, SLOT( dataStateChanged( int ) ) );
 
+  connect( http,   SIGNAL( sslErrors( const QList<QSslError>& ) ),
```

Comment:
Can you please use new style signal connections?

Reply:
It's just a quick and dirty fix, I don't like to update all other connect in a class that has been removed in QGIS 3.

Reply:
It's not supported by Qt4 anyway, new style signals don't work on QGIS 2